src/utils/rocksdb_serialization.hpp

```diff
@@ -30,6 +30,50 @@ namespace memgraph::utils {
 static constexpr const char *outEdgeDirection = "0";
 static constexpr const char *inEdgeDirection = "1";
 
+namespace {
+struct StartEndPositions {
+  size_t start;
+  size_t end;
+
+  size_t Size() const { return end - start; }
+  bool Valid() const { return start != std::string::npos; }
+};
+
+inline StartEndPositions FindPartStartEndPositions(const std::string_view str, const char delim,
```

Comment:
Since this is being called through other functions, could we make the `partNumber` a template?

Reply:
Sorry, I meant doing `template <int partNumber>` but after doing some benchmarks, I don't see any difference, so this could just stay `const int` I guess.